### PR TITLE
Doc: use new URLs in "Astarte in 5 minutes"

### PR DESCRIPTION
**Pull request by Annopaolo**
Use the addresses exposed by Traefik rather than clunky ports.

### CODE REVIEW

**Comment by Annopaolo** [comment]
Based, based on #768 .

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 1]
```diff
@@ -70,18 +70,17 @@ $ docker-compose up -d
 
 `docker-compose-initializer` will generate a root CA for devices, a key pair for Housekeeping, and a self-signed certificate for the broker (note: this is a *really* bad idea in production). You can tune the compose file further to use legitimate certificates and custom keys, but this is out of the scope of this tutorial.
 
-Compose might take some time to bring everything up, but usually within a minute from the containers creation Astarte will be ready. Compose will forward the following ports to your machine:
+Compose might take some time to bring everything up, but usually within a minute from the containers creation Astarte will be ready.
+It will bring up a Traefik instance to expose Astarte services. You can reach Astarte at the following addresses:
```
Comment (Pavinati) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:74]: I wouldn't mention Traefik here. People Reading this guide are focused on trying out Astarte without going too much into how it  works under the hood

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 2]
```diff
@@ -70,18 +70,17 @@ $ docker-compose up -d
...
 
-* `4000`: Realm Management API
-* `4001`: Housekeeping API
-* `4002`: AppEngine API
-* `4003`: Pairing API
-* `4040`: Dashboard
-* `8883`: MQTTS
-* `1885`: MQTT with Proxy Protocol for SSL termination (won't be used)
-* `80`: Let's Encrypt verification (won't be used)
+* `api.localhost`: Astarte API
```
Comment (Pavinati) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:76]: Maybe it's still worth mentioning the services URLs, expecially pairing, considering it's used by stream-test Qt5

File: docker-compose.yml [hunk 3]
```diff
@@ -190,26 +252,31 @@ services:
...
-      # You can comment this out if you don't need Let's Encrypt
-      - "80:80"
-      # HTTP Listener for health check
-      - "8888:8888"
+    # ports:
+    #   # HTTP Listener for health check
+    #   - "8888:8888"
+    #   # 1883 is used internally and isn't exposed
+    #   # Used with HAProxy only
+    #   - "1885:1885"
```
Comment (davidebriani) [inline docker-compose.yml:260]: We can remove this if unused

**Review by davidebriani — APPROVED** [review]

File: doc/pages/tutorials/010-astarte_in_5_minutes.md [hunk 4]
```diff
@@ -183,26 +186,26 @@ $ docker pull astarte/astarte-stream-qt5-test:v1.1.0-alpha.0
...
+Moreover, Astarte's Docker Compose also installs [Astarte Dashboard](https://github.com/astarte-platform/astarte-dashboard), from which you can manage your Realms and install Triggers, Interfaces and more from a Web UI. It is accessible by default at `http://dashboard.localhost` - remember that if you are not exposing Astarte from `localhost`, you have to change Realm Management API's URL in Dashboard's configuration file, to be found in `compose/astarte-dashboard/config.json` in Astarte's repository. You can generate a token for Astarte Dashboard, as usual, through `astartectl utils gen-jwt all-realm-apis -k test_private.pem`. By default, `astartectl` will generate a token valid for 8 hours, but you can set a specific expiration by using the `-e <seconds>` parameter.
```
Comment (Pavinati) [inline doc/pages/tutorials/010-astarte_in_5_minutes.md:208]: `dashboard.astarte.localhost` instead of `http://dashboard.localhost`

**Review by Pavinati — CHANGES_REQUESTED** [review]

**Review by rbino — APPROVED** [review]

**Review by Pavinati — APPROVED** [review]

**Comment by Annopaolo** [comment]
Drafted until #780 is merged.

**Pull request merged**